eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/model/dbsnp/DbsnpClusteredVariantEntity.java

```diff
@@ -17,150 +17,30 @@
  */
 package uk.ac.ebi.eva.accession.core.model.dbsnp;
 
-import org.springframework.data.mongodb.core.index.Indexed;
-import org.springframework.data.mongodb.core.mapping.Document;
-import org.springframework.data.mongodb.core.mapping.Field;
-import uk.ac.ebi.ampt2d.commons.accession.persistence.mongodb.document.AccessionedDocument;
-
-import uk.ac.ebi.eva.accession.core.model.ClusteredVariant;
 import uk.ac.ebi.eva.accession.core.model.IClusteredVariant;
+import uk.ac.ebi.eva.accession.core.model.eva.ClusteredVariantEntity;
 import uk.ac.ebi.eva.commons.core.models.VariantType;
 
 import java.time.LocalDateTime;
 
-@Document
```

Comment:
removed @Document? doesn't need it because the parent has it? have you checked this works?

Reply:
Correct, the parent has it so it is not needed. We are doing it the same way with Submitted Variants